tests/chainer_tests/functions_tests/activation_tests/test_softplus.py

```diff
@@ -15,66 +12,48 @@
     'dtype': [numpy.float16, numpy.float32, numpy.float64],
 }))
 @testing.fix_random()
-class TestSoftplus(unittest.TestCase):
+@testing.inject_backend_tests(
+    None,
+    # CPU tests
+    [
+        {},
+        {'use_ideep': True},
+    ]
+    # GPU tests
+    + testing.product({
+        'use_cuda': [False],
```

Comment:
```suggestion
        'use_cuda': [True],
```

Reply:
Actually marking them True was resulting in several errors in my system.